ares/ws/cartridge/cartridge.cpp

```diff
@@ -48,8 +48,9 @@ auto Cartridge::connect() -> void {
   }
 
   if(auto fp = pak->read("time.rtc")) {
-    rtc.ram.allocate(fp->size());
+    rtc.ram.allocate(18);
```

Comment:
consider leaving this as fp->size, and having mia detect older .rtc files based on size, and re-initializing them to default settings.

Reply:
I honestly have no clue how to do that with Mia's system - there's no documentation that I'm aware of, and VSCodium/clangd not working too well doesn't help browse the codebase...